### ISSUE TITLE
Multiple door factions

### ISSUE THREAD

**Issue by Cannon331** [Chessnut/NutScript#838]
Is there a way to have multiple factions to own a door? Is there coding you have to do to be able to do that?

(Sorry is this is the wrong spot for questions)

**Comment by zoephix** [comment]
Yes ofcourse there's ways, for future questions I recommend joining the NutScript Discord Server. I even created this feature today for myself https://i.gyazo.com/5d3f4212bd58cdd39922072c57a82f39.mp4
